### ISSUE TITLE
WDS and DllPlugin

### ISSUE THREAD

**Issue by alexfedoseev** [webpack/webpack-dev-server#733]
**Do you want to request a *feature* or report a *bug*?**
Bug probably.

**What is the current behavior?**
**If the current behavior is a bug, please provide the steps to reproduce.**

I'm trying to use `DllPlugin` with WDS. We're using WDS with Rails backend and code splitting is done via `CommonsChunkPlugin`. Usually, we load 3 root bundles on the page by injecting `script` tags in the Rails template:

* Vendor libs
* Site navbar bundle
* Site section bundle

Our current development `entry` & `output` look like this:

```js
entry: {
  navbar: [
    'webpack-dev-server/client?http://localhost:3500/',
    'webpack/hot/only-dev-server',
    './path/to/navbar/entry',
  ],
  // N sections
  section: [
    'webpack-dev-server/client?http://localhost:3500/',
    'webpack/hot/only-dev-server',
    './path/to/section/entry',
  ],
  vendor: [ ... ], // vendor libs
},
output: {
  filename: '[name]-bundle.js',
  path: path.resolve(__dirname, 'public'),
  publicPath: `http://localhost:3500/`,
},
```

And server:

```js
const devServer = new WebpackDevServer(compiler, {
  publicPath: webpackConfig.output.publicPath,
  hot: true,
  inline: true,
  historyApiFallback: true,
  quiet: false,
  noInfo: false,
  lazy: false,
  stats: { ... },
});
```

This setup works fine: when I `GET` page, I see one set of messages in console:

```
[HMR] Waiting for update signal from WDS...
[WDS] Hot Module Replacement enabled.
[WDS] Warnings while compiling.
```

But after I switch my setup to DLL build, the output is different.

In short about DLL setup. I precompile DLL build w/ vendor libs and output it in `/dll-build` folder, so as a result, I have compiled `/dll-build/vendor-bundle.js`. When it's done I start WDS w/ the following changes:

* WP config: same `entry` and `output` as above, but without `vendor` entry as it's precompiled
* Server: same opts + added `contentBase: 'dll-build/'` to serve compiled vendor bundle

When I `GET` page, all bundles are loaded, but I see 3 set of messages in the console. I guess each loaded bundle emits one set. Same happening on updates.

**What is the expected behavior?**
One set of messages from WDS (as it works with `CommonsChunkPlugin`).

**Please mention your webpack and Operating System version.**

```
webpack@2.2.0-rc.3
webpack-dev-server@2.2.0-rc.0
macOS@10.11.6
```
